OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/ChapterAdapter.java

```diff
@@ -1,158 +0,0 @@
-package org.edx.mobile.view.adapters;
-
-import android.content.Context;
-import android.os.SystemClock;
-import android.view.View;
-import android.view.View.OnClickListener;
-import android.widget.AdapterView;
-import android.widget.ImageView;
-import android.widget.LinearLayout;
-import android.widget.RelativeLayout;
-import android.widget.TextView;
-
-import org.edx.mobile.R;
-import org.edx.mobile.core.IEdxEnvironment;
-import org.edx.mobile.model.api.SectionEntry;
-import org.edx.mobile.module.db.DataCallback;
-import org.edx.mobile.util.NetworkUtil;
-import org.edx.mobile.view.custom.ProgressWheel;
-
-public abstract class ChapterAdapter extends BaseListAdapter<SectionEntry> {
-
-    private String courseId;
-    private long lastClickTime;
-
-
-    public ChapterAdapter(Context context, String courseId, IEdxEnvironment environment) {
-        super(context, R.layout.row_chapter_list, environment);
```

Comment:
Reverted the deletion commit, it requires to clean all the chapter related code in a separate story because there are handsome database calls related to it and there is a column in DB table as well with name "chatper_name".